www/scripts/codecheckerviewer/hashHelper.js

```diff
@@ -18,11 +20,24 @@ function (hash, ioQuery) {
       delete values.report;
       values.baseline = baselineId;
       values.newcheck = newcheckId;
+      if(values.filters)
+        values.filters = JSON.stringify(filters);
       hash(ioQuery.objectToQuery(values));
     },
 
     setReport : function (reportId) {
       values.report = reportId;
+      if(values.filters)
+        values.filters = JSON.stringify(filters);
+      hash(ioQuery.objectToQuery(values));
+    },
+
+    setFilter : function (filterGroup) {
+      filters = []
+      filterGroup.getChildren().forEach(function (filter) {
```

Comment:
I think you can use the javascript map function here (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map):
```javascript
var filters = filterGroup.getChildren().map(function (filter) {
  return filter.getValues();
});
```